### ISSUE TITLE
In MongoDB "_id" is a custom string field, when trying to transfer data to ElasticSearch getting parse error on "_id" field.

### ISSUE THREAD

**Issue by tincapo** [compose/transporter#73]
MongoDB that I'm using have a custom type "_id" field which is a string representation of GUID such as "21EC2020-3AEA-4069-A2DD-08002B30309D". 
I've setup the application to get the data transferred to ElasticSearch without any transformations and I'm getting an error:
```org.elasticsearch.index.mapper.MapperParsingException: failed to parse [_id].```
The index in ES have "_id" field specified as string.
I've been using River plugin so far and never came across this problem.
So I wonder is it a problem with the way transporter is assuming that this field will be an object or the fault is actually on ES side?

**Comment by Garito** [comment]
org.elasticsearch.index.mapper.MapperParsingException: failed to parse [_id]

**Comment by tincapo** [comment]
Actually that isn't a problem any more when using ES 1.6.0. But still I wonder how to correctly handle custom _id field from MongoDB.

**Comment by codepope** [comment]
Interesting. Has the problem just gone away with 1.6.0 or is there  a specific 1.6.0 change that fixed it for you?

**Comment by tincapo** [comment]
I really don't know TBH. I've tried to sync MongoDB 2.6.3 with ES 1.1.1, 1.5.0 before using exactly the same configs. When I've installed 1.6.0 and started testing I thought I'll give Transporter a go again. I've used what I had prepared already and worked straight away. Makes me think it's ES change that they've introduced in 1.6.0 (I've noticed a lot of fixes around mappings). Still worth sharing if anybody else encountered this.
